docs/jax.ops.rst

```diff
@@ -21,6 +21,7 @@ follows (where ``idx`` is a NumPy index expression).
 =========================  ===================================================
 Alternate syntax           Equivalent in-place expression
 =========================  ===================================================
+``x.at[idx].get(y)``       ``x[idx]``
```

Comment:
I think this should be `x.at[idx].get()`?

Reply:
Thanks! Yes. #8000 fixes it.